src/koans/01_equalities.clj

```diff
@@ -32,5 +32,8 @@
   "Symbolism is all around us"
   (= 'hello (symbol __))
 
+  "Introducing nil"
```

Comment:
No problem! I think the best way would be something along the lines of `What could be equivalent to nothing?`, although that is more indirect in saying what nil is, it gets the point across that nil is used to represent nothing and what it would be equal in a short manner